### DISCUSSION TITLE
How to solve it

### DISCUSSION THREAD

**Discussion by zh758631301** [deepjavalibrary/djl#1713] [Q&A]
![image](https://user-images.githubusercontent.com/49860741/173530902-0b21a0b6-5595-4ccd-9bd5-dd2696983000.png)

The CPU of aarch64 architecture uses modelzoo in api0.17.0 in the integrated djl The following problems occur in the loadmodel (recognition. Detectcriteria()) method. Specifically, execute native in the loadlibrary() method in the cudautils class Error in load ("cudart", cudalibrary.class). Native uses classes in jna5.8.0

**Comment by frankfliu** [comment]
Can you make sure JNA jar file is in your classpath?
